### ISSUE TITLE
Wrong glottocodes

### ISSUE THREAD

**Issue by FredericBlum** [lexibank/lexibank-analysed#66]
Stumbled across those warnings, seems like we still have some wrong glottocodes. I will prepare the minor Pr's and tag accordingly.

```
WARNING Fulfulde / baf2 / maas1242
WARNING Songhai_Senni / baf2 / humb1244
WARNING Sanapaná (Angaité) / johanssonsoundsymbolic / anga1216
WARNING Pubarong Queyu / lairgyalrong / quey1239
WARNING SanJuanDiuxiMixtec / mixtecansubgrouping / diux1236
WARNING SantiagoTilantongoMixtec / mixtecansubgrouping / diux1236**
````

**Comment by FredericBlum** [comment]
@xrotwang Interestingly, those cases did not throw an error during the pylexibank conversion. Somehow I thought such a check would exist, but perhaps we should implement one?

**Comment by FredericBlum** [comment]
Talking about the creation for the individual datasets. During the lexibank_analysed conversion, we did get the above warnings.

**Comment by xrotwang** [comment]
https://github.com/cldf/pycldf/issues/181

**Comment by xrotwang** [comment]
Funny enough, we already have that:
```
$ cldf check ../../cldf-datasets/imtvault/cldf/Generic-metadata.json --glottolog ../../glottolog/glottolog
WARNING Language urad1239 mapped to Bookkeeping languoid urad1239
WARNING Language kang1282 mapped to Bookkeeping languoid kang1282
WARNING Language fumm1238 mapped to Bookkeeping languoid fumm1238
WARNING Language chum1264 mapped to Bookkeeping languoid chum1264
WARNING Language oldp1255 mapped to Bookkeeping languoid oldp1255
WARNING Language bemb1258 mapped to Bookkeeping languoid bemb1258
```
The fact that I had forgotten about this should probably tell me something, though.

**Comment by FredericBlum** [comment]
oh, cool, so i should definitly run this against the Lexibank data. I suspect there _might_ be some further cases... hope it's not too many though.

**Comment by FredericBlum** [comment]
The check from above only finds bookkeeping cases though, and not the non-existing glottocodes. So we should probably refine that.

**Comment by FredericBlum** [comment]
This is now fixed in branch <update-datasets>
